tooling/noir_js/src/program.ts

```diff
@@ -1,13 +1,25 @@
 /* eslint-disable  @typescript-eslint/no-explicit-any */
 import { Backend, CompiledCircuit } from '@noir-lang/types';
 import { generateWitness } from './witness_generation.js';
+import initAbi from '@noir-lang/noirc_abi';
+import initACVM from '@noir-lang/acvm_js';
 
 export class Noir {
   constructor(
     private circuit: CompiledCircuit,
     private backend: Backend,
   ) {}
 
+  async init(): Promise<void> {
```

Comment:
Theres an argument to be made for this to be a standalone method, so that if the Noir class is not used, then there is an initWASMs() method that one can use

Reply:
Oerhaps better to leave it until that initialization is needed, as shown here in [Lazy init](https://github.com/noir-lang/noir/blob/6c1e5caedd934782dd1ecca3bd6f09d552d1fd2e/tooling/noir_js/src/witness_generation.ts#L8)

Reply:
I have added an issue here for it, as to whether we should be lazy initting things: https://github.com/noir-lang/noir/issues/2910#issuecomment-1742811237 @signorecello noted that it seems better to call .init once